pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -60,38 +60,50 @@ func (z *zedrouter) getNIBridgeConfig(
 			Mask: status.Subnet.Mask,
 		}
 	}
+	// TODO: multipath routes
```

Comment:
Are you planning to add multipath routes any time soon? Wouldn't that require EVE API changes if we want static multipath routes from the controller, etc?

Reply:
Yes, I decided to split this into multiple PR otherwise it would probably crash reviewer's web browser :)
In this PR we just add enhancements to support multipath routes and Local NIs with multiple ports into the NI Reconciler component. However, until the upcoming PRs (which I have prepared in my fork), zedrouter will not be taking these new features into the effect (and therefore there will be no changes/improvements seen after we merge this).
In the next PR or the one after it, I will update EVE API used by pillar to include https://github.com/lf-edge/eve-api/pull/53  
(zedagent will parse these new fields and zedrouter will configure NI Reconciler accordingly)